user_tasks/models.py

```diff
@@ -57,6 +72,13 @@ class UserTaskStatus(TimeStampedModel):
     total_steps = models.PositiveSmallIntegerField()
     attempts = models.PositiveSmallIntegerField(default=1, help_text='How many times has execution been attempted?')
 
+    class Meta:
```

Comment:
My understanding is that doing so is only required if the model is not defined in an app included in INSTALLED_APPS: https://docs.djangoproject.com/en/1.10/ref/models/options/#app-label .  This one definitely should be included in that setting, due to migrations and such.